vue-app/src/views/Verify.vue

```diff
@@ -70,19 +70,26 @@
               {{
                 isManuallyVerified
                   ? "You're BrightID verified! Complete the remaining steps to start contributing."
-                  : 'Follow the instructions below to get verified. It’s not immediate so feel free to click the below button when you’re ready.'
+                  : 'Follow the instructions below to get verified in BrightID.'
               }}
             </p>
-            <div class="actions">
-              <button
-                v-if="!isManuallyVerified"
-                @click="handleIsVerifiedClick"
-                class="btn-primary"
-                :disabled="loadingManualVerify"
-              >
-                Check if you are verified
-              </button>
-              <loader v-if="loadingManualVerify" />
+            <div v-if="!isManuallyVerified">
+              <p>
+                Once you're verified in BrightID, verification confirmation
```

Comment:
```suggestion
Verification confirmation
```
This seems a little contradictory saying once your verified check verification. Maybe just seems weird to me so feel free to push back, but could probably just get rid of the first part of this sentence.

Reply:
On second thought, I think this makes sense.